### PR TITLE
Run jest only from the root

### PR DESCRIPTION
**Pull request by TrySound**
This allows to reduce configuration and modules count per each package.

All dependencies (except rebass and styled-system) are reused from the
root. With yarn workspaces all depenendecies could be hoised without
need to remove them from each packages. That's why I removed evenr react
packages.

### CODE REVIEW

**Comment by TrySound** [comment]
@jxnblk I already implemented packages builder locally and gonna send a few pull requests separately starting with this one.

**Comment by TrySound** [comment]
Installing dependencies for each package sucks to be honest. And there are only two packages yet.

**Comment by TrySound** [comment]
@jxnblk Is this look fine?

File: packages/position/test/index.js [hunk 1]
```diff
@@ -1,61 +1,52 @@
...
-      />
-    )
-    expect(json).toHaveStyleRule('position', 'fixed')
-    expect(json).toHaveStyleRule('z-index', '1')
-    expect(json).toHaveStyleRule('top', '0')
-    expect(json).toHaveStyleRule('right', '0')
-    expect(json).toHaveStyleRule('bottom', '0')
-    expect(json).toHaveStyleRule('left', '0')
-  })
+test('Position renders', () => {
```
Comment (jxnblk) [inline packages/position/test/index.js]: Did you mean to remove the `describe`?

Comment (TrySound) [reply packages/position/test/index.js]: I think this is cleaner considering that the file is already scope. `describe` helps with deeper scope like in root tests.

File: packages/hide/test/index.js [hunk 2]
```diff
@@ -1,43 +1,20 @@
-import 'jest-styled-components'
 import React from 'react'
 import TestRenderer from 'react-test-renderer'
 import Hide from '../src'
 
 const renderJSON = el => TestRenderer.create(el).toJSON()
 
-describe('@rebass/hide', () => {
-
-  test.skip('Mapped renders', () => {
```
[the remark this answers is not in the document]
Comment (TrySound) [reply packages/hide/test/index.js]: This test is used not existing component and do not different from the next one.

File: packages/position/test/index.js (hunk 1 above)
Comment (jxnblk) [reply packages/position/test/index.js]: Sure but this change is unrelated to the PR

**Comment by TrySound** [comment]
Rebased

**Comment by TrySound** [comment]
@jxnblk The next PR is ready locally. Will send after merging this.

File: packages/hide/test/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jxnblk) [reply packages/hide/test/index.js]: This is still a placeholder that I want to keep, even if it's not up to date

File: packages/hide/test/index.js [hunk 3]
```diff
@@ -33,11 +15,8 @@ describe('@rebass/hide', () => {
         xlarge
       />
     )
-    console.log('json', json)
+
     expect(json).toMatchSnapshot()
-    expect(json).toHaveStyleRule('display', 'none', {
-      // media: 'screen and (min-width:40em)'
```
Comment (jxnblk) [inline packages/hide/test/index.js]: I think this was maybe a bug with jest-styled-components, would be good to have tests for the media query aspect of this component

Comment (TrySound) [reply packages/hide/test/index.js]: The bug still appears. There is not media queries with any combination. Reverted comment. Not related to this PR.

File: packages/hide/test/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (TrySound) [reply packages/hide/test/index.js]: Reverted.

**Comment by TrySound** [comment]
@jxnblk Is this ok for you?

**Comment by TrySound** [comment]
Friendly ping @jxnblk

**Comment by TrySound** [comment]
Ping @jxnblk

**Comment by jxnblk** [comment]
Thanks for this! And sorry haven't done any work on Rebass lately

**Pull request merged**